LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019 Makeability Lab
```

Comment:
It would be nice to add a note to the top to say that this is an MIT license. Similar to how we do in core sidewalk: https://github.com/ProjectSidewalk/SidewalkWebpage/blob/master/LICENSE.md